src/compas/geometry/brep/__init__.py

```diff
@@ -72,6 +72,11 @@ def from_pipe(*args, **kwargs):
     raise PluginNotInstalledError
 
 
+@pluggable(category="factories")
+def from_plane(*args, **kwargs):
+    raise PluginNotInstalledError
+
```

Comment:
Is there a rhino implementation that we could add right away as well? Or will this be only OCC?

Reply:
dunno. will check

Reply:
no, Rhino doesn't have a `CreateFromPlane` or similar...